modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/notification-templates/cloud-next-steps-template/en-US.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta content="IE=edge" http-equiv="X-UA-Compatible" />
+		<meta content="width=device-width, initial-scale=1.0" name="viewport" />
+
+		<title>Document</title>
+	</head>
+<div style=" display: flex; flex-direction: column; justify-content: center; align-items: center; font-family: 'Source Sans Pro', sans-serif; ">
```

Comment:
Please, format this code. It needs to be aligned.

Reply:
Done.